lib/finishm/orfs_finder.rb

```diff
@@ -31,6 +34,15 @@ def validate_options(options, argv)
       visualise.validate_assembly_options(options)
   end
```

Comment:
be good to validate the options for output too - need one or more of fna and faa yeh?